tests/test_workspace.py

```diff
@@ -116,6 +124,8 @@ def test_workspace_gcs_connection():
         s3_connection = workspace.gcs_connection(identifier=identifier)
         assert s3_connection.service_account_key == service_account_key
         assert s3_connection.bucket_name == bucket_name
+        assert re.search("service_account_key", repr(s3_connection)) is None
+        assert re.search("service_account_key", str(s3_connection)) is None
 
 
 def test_workspace_custom_connection():
```

Comment:
Would it be possible to have a test for secret values in custom connection as well?

Reply:
Okay